sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=org.sonarqube:cvs-tsk-report-gen
+
+# this is the name and version displayed in the SonarQube UI. Was mandatory prior to SonarQube 6.1.
+sonar.projectName=Report Generation Microservice
+sonar.projectVersion=1.0
+
+# Path is relative to the sonar-project.properties file. Replace “\” by “/” on Windows.
+# This property is optional if sonar.modules is set.
+sonar.sources=src
+
+sonar.exclusions=.build, src/models/**, src/config/**, src/models/**, src/services/LambdaService.ts, src/services/S3BucketService.ts, src/handler.ts, src/functions/**
```

Comment:
functions should not be excluded

Reply:
The functions cannot be unit tested, and do not contain core logic. All core logic has been tested